pkg/pipeline/sink/uploader/s3.go

```diff
@@ -128,6 +129,12 @@ func newS3Uploader(conf *config.EgressS3Upload) (uploader, error) {
 
 	if conf.Tagging != "" {
 		u.tagging = aws.String(conf.Tagging)
+	} else {
+		u.tagging = aws.String("inline")
```

Comment:
what does `inline` do?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition

it does seem like attachment would be a better default.. but changing it may be disruptive to users.